single-node.sh

```diff
@@ -35,6 +35,13 @@ cat ${HOME}/.ssh/id_rsa.pub >> ${HOME}/.ssh/authorized_keys
 
 runfabtests_script="${HOME}/libfabric/fabtests/install/bin/runfabtests.sh"
 
+EXCLUDE=${HOME}/libfabric/fabtests/install/share/fabtests/test_configs/${PROVIDER}/${PROVIDER}.exclude
```

Comment:
Considering the efa.exclude is already modified by `install-fabtests.sh` (exclude `dgram_pingpong`), do we need to run `dgram_pingpong` explicitly with `-b` like what we did in multi-node test?

Reply:
I do not think running single node dgram_pingpong give us more data. @rwespetal what do you think?

Reply:
It's fine with me to skip the loopback dgram_pingpong test.

Reply:
Even if we want single node dgram-pingpong test, that should be a separate PR. This patch is a bug fix,

Reply:
I see, thanks!